### PR TITLE
Fix df_set_timeout loop event issue

### PR DESCRIPTION
**Pull request by bhardwajaditya**
Closes https://github.com/WideChat/Rocket.Chat/issues/940

### CODE REVIEW

**Comment by ear-dev** [comment]
@Shailesh351 can you review and test this please?  thanks......

File: lib/EventTimeoutProcessor.ts [hunk 1]
```diff
@@ -1,30 +1,37 @@
...
         const event = { name: jobContext.eventName, languageCode: data.custom_languageCode || defaultLanguageCode || LanguageCode.EN, parameters: {} };
-        const response = await Dialogflow.sendRequest(http, read, modify, jobContext.rid, event, DialogflowRequestType.EVENT);
+        const response: IDialogflowMessage = (await Dialogflow.sendRequest(http, read, modify, jobContext.rid, event, DialogflowRequestType.EVENT));
+
+        const room = await read.getRoomReader().getById(jobContext.rid) as ILivechatRoom;
+        if (!room || !room.isOpen) { throw new Error('Error! Invalid session Id. No active room found with the given session id'); }
+        const { visitor: { token: visitorToken } } = room;
 
-        await createDialogflowMessage(jobContext.rid, read, modify, response);
+        await handleResponse(this.app, read, modify, http, persis, jobContext.rid, visitorToken, response );
```
Comment (Shailesh351) [inline lib/EventTimeoutProcessor.ts:34]: `jobContext.rid` is being used multiple times in this function. We should create `const` and use it.

Functionally It's looking good 👍

Comment (bhardwajaditya) [reply lib/EventTimeoutProcessor.ts]: Done 👍

**Review by ear-dev — APPROVED** [review]
Thanks for the quick turn around!

**Pull request merged**